### PR TITLE
Fix Azure CI

### PR DESCRIPTION
**Pull request by astrofrog**
Work in progress - for some reason the Azure CI was broken and wasn't reporting on PRs.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@ pr:
 variables:
   CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
   CIBW_MANYLINUX_I686_IMAGE: manylinux1
+  CI: true
```
Comment (pllim) [inline azure-pipelines.yml:34]: This is also in #11089 , which is milestoned for 4.2.1. FYI.

Comment (astrofrog) [reply azure-pipelines.yml:34]: Ah thanks!

**Comment by pllim** [comment]
Hmm... OSX failure is from `hypothesis` and Windows is from some linking problem in `wcsapi` C-extension.

**Comment by pllim** [comment]
Maybe we don't need to run hypothesis for wheel tests...? Is there a way to turn it off?

File: azure-pipelines.yml (hunk 1 above)
Comment (astrofrog) [reply azure-pipelines.yml:34]: I've now backported that PR correctly and rebased this PR

**Comment by astrofrog** [comment]
I've now backported #11089 and #11401 and rebased this

**Comment by pllim** [comment]
I think we can ignore mpldev failure.

**Review by pllim — APPROVED** [review]
Azure is green, so I guess LGTM.

**Pull request merged**